package.json

```diff
@@ -84,19 +84,20 @@
     "builtin-modules": "3.1.0",
     "codecov": "3.6.1",
     "cross-env": "6.0.3",
+    "cspell": "4.0.44",
+    "eslint": "6.6.0",
     "eslint-config-prettier": "6.7.0",
     "eslint-formatter-friendly": "7.0.0",
     "eslint-plugin-import": "2.19.1",
     "eslint-plugin-prettier": "3.1.2",
     "eslint-plugin-react": "7.17.0",
     "eslint-plugin-unicorn": "13.0.0",
-    "eslint": "6.6.0",
     "execa": "3.4.0",
+    "jest": "24.9.0",
```

Comment:
What caused this reordering?

Reply:
@thorn0 next branch deps not sorted. every time install new deps will reorder.